src/components/Task/Drawer/CompleteForm/CompleteForm.tsx

```diff
@@ -94,7 +95,22 @@ const TaskDrawerCompleteForm = ({
   const [updateTask, { loading: saving }] = useCompleteTaskMutation();
   const onSubmit = async (attributes: TaskUpdateInput): Promise<void> => {
     try {
-      await updateTask({ variables: { accountListId, attributes } });
+      const endOfDay = DateTime.local().endOf('day');
+      await updateTask({
+        variables: { accountListId, attributes },
+        refetchQueries: [
+          {
+            query: GetThisWeekDocument,
+            variables: {
+              accountListId,
+              endOfDay: endOfDay.toISO(),
+              today: endOfDay.toISODate(),
+              twoWeeksFromNow: endOfDay.plus({ weeks: 2 }).toISODate(),
+              twoWeeksAgo: endOfDay.minus({ weeks: 2 }).toISODate(),
+            },
+          },
+        ],
```

Comment:
I had to use ```refetchQueries``` here again for the same reason as last time. The user can now complete a task on the dashboard and with the amount of data from the ```GetThisWeek``` query, it's not easy to update the cache correctly.